source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,462 @@
+.. _minio-site-replication-overview:
+
+=========================
+Site replication overview
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Site replication configures multiple independent MinIO sites or clusters as replicas.
+When configured this way, refer to the replica sites as either "peer sites" or "sites".
+
+Site replication assumes the use of an external identity provider (IDP).
+All configured deployments must use the same external IDP.
+
+
+What replicates across all sites?
+---------------------------------
+
+When enabling site replication on a set of sites, the following changes apply to the other sites:
+
+- Creation and deletion of buckets and objects, including
+
+  - Configuration
+  - Policy changes and deletions
+  - Tags
+  - Locks
+  - Encryption settings
+
+- Creation and deletion of IAM users, groups, policies, and policy mappings to users or groups (for LDAP users or groups)
+- Creation of Security Token Service (STS) credentials for session tokens verifiable from the local ``root`` credentials
+- Creation and deletion of service accounts (except those owned by the ``root`` user)
+- Changes to bucket features, such as
+  
+  - :ref:`Policies <minio-policy>`
+  - Tags (for example, see :mc-cmd:`mc tag set`)
+  - :ref:`Object-Lock <minio-object-locking>` configurations (including retention and legal hold configurations)
+  - :ref:`Encryption configuration <minio-encryption-overview>`
+
+Site replication enables bucket versioning for all new and existing buckets on all replicated sites.
+
+
+What doesn't replicate across sites?
+-------------------------------------
+
+The following bucket features don't replicate across sites set up for site replication:
+
+- Notification configurations
+- Lifecycle management (ILM) configurations
+
+This allows users who access different sites to define separate notification or ILM settings.
+
+
+Order of operations
+-------------------
+
+After enabling site replication, identity and access management (IAM) settings sync in the following order:
+
+#. Policies
+#. User accounts (for local users)
+   
+   Note: Sites that use OIDC and LDAP don't sync, and they may not have local users.
+
+#. Groups 
+   
+   For MinIO IDP and LDAP managed groups
+
+#. Service accounts
+
+   Only static accounts with valid claims.
+   Service accounts for ``root`` don't sync.
+
+#. Policy mapping for synced user accounts
+
+#. Policy mapping for `Short Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+Site healing
+------------
+
+With site replication enabled, bucket metadata and IAM entries can heal from whichever peer site has the most updated entry.
```

Comment:
I assume this is true in general? For any of the items which replication supports (identified above), the MinIO deployment resynchronizes that item from the remote replica with the most updated ("latest") version of that item.